### PR TITLE
ci: fix node events on cypress preset

### PR DESCRIPTION
**Pull request by johnhwhite**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1824/storybook/)

Component Storybooks:

- _(no component storybooks affected in this pr)_ 

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1824/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1824/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1824/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1824/playground/)

File: apps/integration-e2e/src/e2e/lookup-in-modal.cy.ts [hunk 1]
```diff
@@ -42,7 +42,6 @@ describe('lookup in modal', () => {
               });
           });
         cy.window().skyVisualTest(`lookup-in-modal-${theme}`, {
-          capture: 'viewport',
```
Comment (johnhwhite) [inline apps/integration-e2e/src/e2e/lookup-in-modal.cy.ts:45]: The viewport scrollbars were showing in the screenshot when this was enabled.

File: libs/sdk/e2e-schematics/src/cypress-preset.ts [hunk 2]
```diff
@@ -21,6 +21,7 @@ export function skyE2ePreset(
       runMode: 2,
       openMode: 0,
     },
+    ...options,
```
Comment (johnhwhite) [inline libs/sdk/e2e-schematics/src/cypress-preset.ts:24]: Launch options weren't being applied if `options` contained `setupNodeEvents`.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick successful! :cherries: :tada: See https://github.com/blackbaud/skyux/pull/1825

**Pull request merged**